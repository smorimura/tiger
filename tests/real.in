0;
0.0;
0.1;
100;
100.0;
100.1;