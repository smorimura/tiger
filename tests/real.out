0
0
0.1
100
100
100.1
